crt_portal/static/js/other_show_hide.js

```diff
@@ -1,22 +1,26 @@
-var otherbox = document.getElementById('other-class-option');
-otherbox.setAttribute('hidden', '');
+(function(dom) {
+  /**
+   * The index of the 'other' option on the protected class form.
+   * We target it explicitly becuase while it is currently
+   * the last element in the list of options, it might not always be
```

Comment:
Do we want to warn people to keep the other option as last?

If we don't have other as the last option, we will have to reimplement how we add the other option text and I think it would be hard to have a good no javascript fall back for it. If it is always at the end, it is easy for the non-collapsed version to be generated.

Reply:
So, you think we should amend this comment to specifically say that the `other` option should always be last? That warning might be a good idea, but I don't think it should live in the JS file. It think it could be co-located with python code?

Reply:
sounds good I can put in a small PR for that.